spec/support/shared_examples/mappings_batch.rb

```diff
@@ -1,3 +1,34 @@
+shared_examples 'creates redirect mapping' do
+  context 'some context' do
```

Comment:
Yep, @rgarner, I tried to explain this, possibly not very well, in the commit messages for those changes (last line in the message) https://github.com/alphagov/transition/commit/36461a3b88873e0447101757a42867c01361f589